broker/src/test/java/io/camunda/zeebe/broker/jobstream/StreamRegistryTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.jobstream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.atomix.cluster.MemberId;
+import io.camunda.zeebe.broker.jobstream.ImmutableStreamRegistry.StreamConsumer;
+import io.camunda.zeebe.broker.jobstream.ImmutableStreamRegistry.StreamId;
+import io.camunda.zeebe.util.buffer.BufferUtil;
+import java.util.UUID;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.junit.jupiter.api.Test;
+
+class StreamRegistryTest {
+
+  final StreamRegistry<Integer> streamRegistry = new StreamRegistry<>();
+  final MemberId gateway = MemberId.from("gateway");
+  final MemberId otherGateway = MemberId.from("gateway-other");
+  final UnsafeBuffer typeBar = new UnsafeBuffer(BufferUtil.wrapString("bar"));
```

Comment:
:upside_down_face: Can be `var`, since from the constructor we basically know the type already